capellambse/model/__init__.py

```diff
@@ -339,3 +339,7 @@ def search(
     def by_uuid(self, uuid: str) -> common.GenericElement:
         """Search the entire model for an element with the given UUID."""
         return common.GenericElement.from_model(self, self._loader[uuid])
+
+    def __contains__(self, obj: common.GenericElement) -> bool:
+        assert isinstance(obj, common.GenericElement)
+        return obj._model is self
```

Comment:
Just an idea: Since I don't want to call `obj._model is model` in the tests. I recognized we don't yet have a quick check for this.

Reply:
Why would we need to check that `obj._model is model` though? What's the relevant bit in the tests that you're talking about?